app/static/js/spcconvert.js

```diff
@@ -446,25 +446,40 @@ function setGtruth() {
 // show incorrect image indexed at img_i
 function showImg(img_i) {
     // insert image
+    console.log(incorrectArr[img_i]);
     $("#annot-img").attr("src", "static/" + incorrectArr[img_i]);
     
     // populate information
     img = roistore({url: incorrectArr[img_i]}).get();
+    img = img[0];
 
     // Compute Image size from ImageDetail Container
     var aspectRatio = img.image_height/img.image_width;
     var heightScale = $('#annot-img').height()/img.image_height;
-    var widthScale = $('#annot-img').width()/data.image_width;
-    if (data.image_width*heightScale > 0.6*$('#annot-img').width()) {
+    var widthScale = $('#annot-img').width()/img.image_width;
+    if (img.image_width*heightScale > 0.6*$('#TargetImg-annot').width()) {
         // Scale Image by width
-        $('#TargetImg').width(0.6*$('#annot-img').width());
-        $('#TargetImg').height(0.6*$('#annot-img').width()*aspectRatio);
+        $('#TargetImg-annot').width(0.6*$('#TargetImg-annot').width());
+        $('#TargetImg-annot').height(0.6*$('#TargetImg-annot').width()*aspectRatio);
     }
     else {
-        $('#TargetImg').height(0.75*$('#annot-img').height());
-        $('#TargetImg').width(0.75*$('#annot-img').height()/aspectRatio);
+        $('#TargetImg-annot').height(0.75*$('#TargetImg-annot').height());
+        $('#TargetImg-annot').width(0.75*$('#TargetImg-annot').height()/aspectRatio);
     }
 
+    $('#TargetImg-annot').attr("src", "static/" +incorrectArr[img_i]);
+    $('#ImageDetailTitle-annot').html(incorrectArr[img_i]);
+    $('#ImageName-annot').html("<span class='info-label'>Image ID</span>" + "<span>" + img.image_id + "</span>");
+    $('#Timestamp-annot').html("<span class='info-label'>Collection Datetime</span>" + "<span>" + img.image_timestamp + "</span>");
+    // Info
+    var classes = {};
```

Comment:
I'm assuming these are labels for when we label right? Are they still being used (i.e. how come the other 20 classes aren't listed)?

Reply:
These are just to show the prediction probabilities given by the classifier in the sidebar. More classes will be added as we expand the classifier.